string TEMPERATUR_SENSOR = "TEMPERATUR_SENSOR"
string HUMIDITY_SENSOR = "HUMIDITY_SENSOR"
string RF233 = "RF233"

string DATA_TYPE_STRING = "str"
string DATA_TYPE_INT = "int"
string DATA_TYPE_FLOAT = "float"
string DATA_TYPE_JSON = "json"

string name
string type
string guid
string subid

string data
string data_type
string time
